templates/model/spec_test.go

```diff
@@ -378,25 +378,23 @@ params: `,
 			wantValidateErr: `at spec.yaml line 1 column 1: field "message" is required`,
 		},
 		{
-			name: "include_success",
```

Comment:
Instead of replacing the original include_success test, can we have include_old_style as a separate, additional test?

Reply:
Sure, done. Added another test too that just occurred to me.